addons/im_livechat/static/src/models/public_livechat_global.js

```diff
@@ -28,6 +29,12 @@ registerModel({
         },
     },
     recordMethods: {
+        async loadQWebTemplate() {
+            const templates = await this.messaging.rpc({ route: '/im_livechat/load_templates' });
+            for (let template of templates) {
+                qweb.add_template(template);
```

Comment:
```suggestion
                QWeb.add_template(template);
```

Reply:
No, it follows the import, `qweb !== QWeb`. Previous code introduced an unnecessary local variable `QWeb`, so I removed it and just used the actual named export stuff